applications/models.py

```diff
@@ -294,6 +309,18 @@ class ApplicationRoundBasket(models.Model):
         verbose_name=_("Order number"), default=1, null=False, blank=True
     )
 
+    def get_application_events_in_basket(self, applicationevents):
+        events = ApplicationEvent.objects.filter()
```

Comment:
this includes all the ApplicationEvents. Should it be just the ones in the basket?

Reply:
We don't know which events are in the basket since it's purely based on the filtering by purpose, age group etc.

Reply:
But yeah, actually there is a bug here! Should restrict by application round.